Makefile

```
@@ -4,7 +4,7 @@
 
 .PHONY: all help system-setup install-homebrew install-apps install-deb-packages install-flatpak-packages \
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)